charts/tempo-distributed/templates/distributor/service-distributor.yaml

```diff
@@ -1,48 +1,68 @@
 apiVersion: v1
 kind: Service
 metadata:
-  name: {{ template "tempo.fullname" . }}
+  name: {{ template "tempo.fullname" . }}-distributor
```

Comment:
we may want to mention this as a breaking change since it would change how people reference ingestion

Reply:
Added to the README.